variables.tf

```diff
@@ -28,4 +28,29 @@ variable "endpoint_type" {
   }
 }
 
+variable "create_access_group" {
+  type        = bool
+  description = "Whether to create an access group for the secrets group."
+  default     = true
+}
+
+variable "access_group_name" {
+  type        = string
+  description = "Whether to create an access group for the secrets group."
```

Comment:
description needs to be update. Make sure to include what happens if null is passed